### PR TITLE
Merging collapsible if statements increases the code's readability

### PR DESCRIPTION
**Pull request by AymanDF**
This pull request is focused on resolving occurrences of Sonar rule squid:S1066 - “Collapsible "if" statements should be merged”. You can find more information about the issue here: 
https://dev.eclipse.org/sonar/rules/show/squid:S1066
Please let me know if you have any questions.
Ayman Abdelghany.

### CODE REVIEW

File: components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkflowManagementServiceImpl.java [hunk 1]
```diff
@@ -151,10 +151,8 @@ public Workflow getWorkflow(String workflowId) throws WorkflowException {
             }
         }
         for (WorkflowListener workflowListener : workflowListenerList) {
-            if (workflowListener.isEnable()) {
-                if (workflowListener.isEnable()) {
-                    workflowListener.doPostListWorkflowEvents(eventList);
-                }
+            if (workflowListener.isEnable() && workflowListener.isEnable()) {
```
Comment (cdwijayarathna) [inline components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkflowManagementServiceImpl.java]: Seems to be same condition has repeated in both if clauses unnecessarily in previous code, can u remove 1 of them?

**Comment by cdwijayarathna** [comment]
Thanks! fix looks ok, I have suggested one more improvement which was a mistake in existing code, can you fix it as well so that we can go ahead and merge this?

File: components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkflowManagementServiceImpl.java (hunk 1 above)
Comment (AymanDF) [inline components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkflowManagementServiceImpl.java]: @cdwijayarathna Redundant condition has been removed.

**Comment by JKAUSHALYA** [comment]
Closing until conflicts are resolved.

**Pull request closed without merging**